src/components/GordonConfetti/index.jsx

```diff
@@ -22,8 +22,8 @@ const GordonConfetti = ({ active, colorOption, colors }) => {
       '#ddd',
       gordonColors.primary.blue,
       gordonColors.primary.cyan,
-      gordonColors.secondary.orange,
-      gordonColors.secondary.green,
+      //gordonColors.secondary.orange,    //these are commented out so the confetti is Gordon's primary colors
+      //gordonColors.secondary.green,
```

Comment:
Hey @jhammond4 -- there is a configuration setting that you can pass in which will make the confetti be only Gordon primary colors. See lines 28-30.

Please don't change the global Confetti configuration since it affects the Happy Birthday message as well.

Reply:
You can pass in `colors` or `colorOption` to the GordonConfetti component and it will use the colors that you specify.

Reply:
Got it-I can switch this back as well.